package.json

```diff
@@ -1,6 +1,16 @@
 {
   "name": "strong-mesh-client",
   "version": "2.0.1",
+  "keywords": [
+    "strongloop",
+    "mesh"
+  ],
```

Comment:
and if they are there, someone might start asking that they be maintained... like changing strongloop to IBM, so better not to have, IMO